deploy/docs/Installation_with_Helm.md

```diff
@@ -36,66 +41,84 @@ These steps require that no Prometheus exists. If you already have Prometheus in
 - [How to install if you have standalone Prometheus](./standAlonePrometheus.md) 
 - [How to install our Prometheus side by side with your existing Prometheus](./SideBySidePrometheus.md)
 
+The Helm chart installation requires two parameter overrides:
+* __sumologic.accessId__ - Sumo [Access ID](https://help.sumologic.com/Manage/Security/Access-Keys).
+* __sumologic.accessKey__ - Sumo [Access key](https://help.sumologic.com/Manage/Security/Access-Keys).
+
+The following parameter is optional, but we recommend setting it.
+* __sumologic.clusterName__ - An identifier for your Kubernetes cluster.  This is the name you will see for the cluster in Sumo Logic. Default is `kubernetes`.
+
 To install the chart, first add the `sumologic` private repo:
 
 ```bash
 helm repo add sumologic https://sumologic.github.io/sumologic-kubernetes-collection
 ```
 
-Install the chart with release name `collection` and namespace `sumologic`
+Next you can run `helm upgrade --install` to install or upgrade our chart.  An example command with the minimum parameters is provided below for Helm2 and Helm3.
 
+Helm2
 ```bash
-helm install sumologic/sumologic --name collection --namespace sumologic --set sumologic.accessId=<SUMO_ACCESS_ID> --set sumologic.accessKey=<SUMO_ACCESS_KEY>  --set sumologic.clusterName="<MY_CLUSTER_NAME>"
+helm upgrade --install sumologic/sumologic --name my-release --set sumologic.accessId=<SUMO_ACCESS_ID> --set sumologic.accessKey=<SUMO_ACCESS_KEY>  --set sumologic.clusterName="<MY_CLUSTER_NAME>"
```

Comment:
why `my-release` and not `collection`?

Reply:
The reason I switched to this is that I want to make it clear that we no longer require a specific release name.  `my-release` is the default that all helm charts in helm stable provided. I'm not married to it though if we want to change it back.  Since we used to require it to be `collection` and in the `sumologic` namespace, it has led to some confusion on if we still do.